src/map/utils/battleutils.cpp

```diff
@@ -2020,7 +2020,7 @@ namespace battleutils
             if (giveTPtoVictim)
             {
                 //account for attacker's subtle blow which reduces the baseTP gain for the defender
-                float sBlowMult = ((100.0f - dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW), 0.0f, 50.0f)) / 100.0f);
+                float sBlowMult = ((100.0f - (dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW), 0.0f, 50.0f) + dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW_II), 0.0f, 25.0f))) / 100.0f);
```

Comment:
Is Subtle Blow II supposed to cap at +25? As in, you can't do something like, 40 Subtle Blow + 35 Subtle Blow II to hit the limit of 75? Same deal later on in this file, line 2141.